standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/events/CreateTableEvent.java

```diff
@@ -29,11 +30,14 @@
 
   private final Table table;
   private final boolean isReplicated;
+  private final SQLAllTableConstraints constraints;
 
-  public CreateTableEvent(Table table, boolean status, IHMSHandler handler, boolean isReplicated) {
+  public CreateTableEvent(Table table, boolean status, IHMSHandler handler, boolean isReplicated,
+      SQLAllTableConstraints constraints) {
```

Comment:
We have separate events for add individual constraints. I think, we need not keep it as part of create table event. 
It adds additional overhead when transmit events for incremental replication. Also, takes additional storage in HMS.

Reply:
Removed